code/index.html

```diff
@@ -4,18 +4,29 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <link rel="stylesheet" href="style.css" />
-    <title>Technigo Project Cluedo</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="icon" href="./assets/icons8-magnifying-glass.png" type="image/gif">
+    <title>Project Cluedo</title>
   </head>
   <body>
+    <header>
+      <div class="header-div">
+          <p class="header-text">reveal the CLUEDO mystery</p> 
+          <p class="header-text">who killed Mr Black?</p>
+          <p class="header-text">where?</p>
+          <p class="header-text">and how?</p>
+      </div>
+    </header>
+    <div class="instruction-text">Select a card from every deck first, then click on reveal the crime button.</div>
```

Comment:
Really easy to follow/read your code everywhere! 👏 You have class-names that are short and informational, as well as comments in the JS, so that I easily can follow and understand. Nice structure, it looks profesional and clean!

Reply:
Also like how you introduce the user whith "reveal the CLUEDO mystery", "who killed Mr Black?", "ehere?", "and how", and information about select card from every deck... and then click on the....